### PR TITLE
docs(changelog): 5.4 release

### PR DESCRIPTION
**Pull request by P0lip**
**Checklist**

- [x] Tests added / updated
- [x] Docs added / updated

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,27 @@ All notable changes to this project will be documented in this file.
...
+### Changed
+- Rename `oas3-example-value-or-externalValue` rule to `oas3-examples-value-or-externalValue` [#1098](https://github.com/stoplightio/spectral/pull/1098)
+- Drop useless oas rule tags [#1095](https://github.com/stoplightio/spectral/pull/1095)
+- AJV: swallow missing references [#1147](https://github.com/stoplightio/spectral/pull/1147)
+- Improve ruleset validation [#1144](https://github.com/stoplightio/spectral/pull/1144)
+- Allow running spectral from any directory in Docker [#1022](https://github.com/stoplightio/spectral/pull/1022)
+
+### Fixed
+- Improve `oas2-schema` and `oas3-schema` rules [#403](https://github.com/stoplightio/spectral/issues/403)
+- Make `example-value-or-externalValue` only target oas3 document [#883](https://github.com/stoplightio/spectral/issues/883)
```
Comment (nulltoken) [inline CHANGELOG.md:26]: I'd drop this from here and update the `Changed` section to 

```
Drop `example-value-or-externalValue` rule and introduce `oas3-examples-value-or-externalValue` [#883](https://github.com/stoplightio/spectral/issues/883) and [#1098](https://github.com/stoplightio/spectral/pull/1098)
```

From the overall release standpoint, we haven't fixed `example-value-or-externalValue`, we've nuked it ;-)

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,27 @@ All notable changes to this project will be documented in this file.
...
+- `path-params` rule detects undefined parameters across operations [#1076](https://github.com/stoplightio/spectral/issues/1076)
```
Comment (nulltoken) [inline CHANGELOG.md:27]:
```suggestion
- `path-params` rule now properly detects undefined parameters across operations [#1076](https://github.com/stoplightio/spectral/issues/1076)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,27 @@ All notable changes to this project will be documented in this file.
...
+- Load custom functions from NPM correctly [#1093](https://github.com/stoplightio/spectral/pull/1093)
+- `path-params` reports an actual endpoint path [#1130](https://github.com/stoplightio/spectral/issues/1130)
```
Comment (nulltoken) [inline CHANGELOG.md:29]:
```suggestion
- Teach `path-params` to report an actual endpoint path [#1130](https://github.com/stoplightio/spectral/issues/1130)
```

File: CHANGELOG.md (hunk 1 above)
Comment (P0lip) [reply CHANGELOG.md:26]: [keep a changelog](https://keepachangelog.com/en/1.0.0/) has the 'removed' section.
Seems like a perfect fit here.

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,26 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.4.0] - 2019-05-26
```
Comment (P0lip) [inline CHANGELOG.md:10]:
```suggestion
## [5.4.0] - 2019-05-27
```

**Pull request merged**